src/Foundatio.AzureServiceBus/Foundatio.AzureServiceBus.csproj

```
@@ -5,6 +5,10 @@
     <PackageReference Include="Microsoft.Azure.ServiceBus" Version="5.2.0" />
 
     <PackageReference Include="Foundatio" Version="10.7.1" Condition="'$(ReferenceFoundatioSource)' == '' OR '$(ReferenceFoundatioSource)' == 'false'" />
+
+    <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="2.3.24" />
```

```suggestion
    <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="(2.3.24,)" />
```

Maybe to allow future versions but at least get up to this version?